packages/mds-provider-service/repository/index.ts

```diff
@@ -53,11 +53,17 @@ export const ProviderRepository = ReadWriteRepository.Create(
       }
     }
 
-    const getProviders = async (): Promise<ProviderDomainModel[]> => {
+    const getProviders = async (options?: GetProvidersOptions): Promise<ProviderDomainModel[]> => {
```

Comment:
This one doesn't need to be optional since you're passing in `options ?? {}` from the service.